libvast/src/schema.cpp

```diff
@@ -238,11 +238,11 @@ get_schema_dirs(const caf::actor_system_config& cfg,
       VAST_ERROR("{} failed to get program path", __func__);
   }
   if (!disable_default_config_dirs) {
-    result.insert(std::filesystem::path{VAST_SYSCONFDIR} / "vast" / "schema");
-    if (auto xdg_config_home = detail::locked_getenv("XDG_CONFIG_HOME"))
+    result.insert(std::filesystem::path{VAST_CONFIGDIR} / "schema");
+    if (auto xdg_config_home = detail::env("XDG_CONFIG_HOME"))
```

Comment:
`detail::locked_getenv` is reverted to `detail::env` in this commit. Probably unintentional?

Reply:
Yes, rebase mistake. Will fix in a second.